openshift.spec

```diff
@@ -275,7 +411,171 @@ fi
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/policy.json
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/serviceaccounts.private.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/serviceaccounts.public.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/admin.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/admin.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/admin.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/ca.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/ca.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/ca.serial.txt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/etcd.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/etcd.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master-config.yaml
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.etcd-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.etcd-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.kubelet-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.kubelet-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/master.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-master.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-registry.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/openshift-router.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/policy.json
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/serviceaccounts.private.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/serviceaccounts.public.key
+
+%post -n atomic-enterprise-master
+%systemd_post %{basename:atomic-enterprise-master.service}
+%systemd_post %{basename:atomic-enterprise-allinone.service}
```

Comment:
I'll switch it back to master only. All-in-one probably not what people want most of the time.